source/_sass/infeedl.scss

```diff
@@ -63,9 +63,16 @@ h1, h2, h3, h4, h5, h6 {
   li {
     a {
       text-transform: none;
-      color: #fff;
+      color: #F5A623;
       font-size: .9em;
       font-weight: normal;
+      border-bottom:1px solid rgba(251,192,57,.5);
+      border-bottom-style: inset;
+
+      &:hover{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space